Libraries/src/Amazon.Lambda.AspNetCoreServer/AbstractAspNetCoreFunction.cs

```diff
@@ -206,60 +206,6 @@ public void RegisterResponseContentEncodingForContentEncoding(string contentEnco
         /// <param name="builder"></param>
         protected virtual void Init(IWebHostBuilder builder) { }
 
-        /// <summary>
-        /// Creates the IWebHostBuilder similar to WebHost.CreateDefaultBuilder but replacing the registration of the Kestrel web server with a 
-        /// registration for Lambda.
-        /// </summary>
-        /// <returns></returns>
-        [Obsolete("Functions should migrate to CreateHostBuilder and use IHostBuilder to setup their ASP.NET Core application. In a future major version update of this library support for IWebHostBuilder will be removed for non .NET Core 2.1 Lambda functions.")]
```

Comment:
This obsolete method was deleted because it was here to support .NET Core 2.1. The message even said we would delete in the future. The future is now.